ckan/logic/action/create.py

```diff
@@ -621,6 +624,8 @@ def member_create(context: Context,
                 capacity != u'admin':
             raise NotAuthorized("Administrators cannot revoke their "
                                 "own admin status")
+        if member.state != u'active':
+            member.state = u'active'
```

Comment:
```suggestion
        if member.state != 'active':
            member.state = 'active'
```

Can you commit this to trigger a new test run? I can't do it from the UI, thanks @JVickery-TBS